
try{
    let a= 5
   sum = a+b
}catch(err){
    console.log(err);
    console.log("object");
}